src/custom-types.json

```diff
@@ -476,6 +476,15 @@
   "audio/l24": {
     "compressible": false
   },
+  "audio/mobile-xmf": {
+    "extensions":["xmf", "mxmf"],
+    "sources": [
+      "http://tools.ietf.org/rfc/rfc4723.txt",
+      "http://www.iana.org/assignments/media-types/audio/mobile-xmf",
+      "https://filext.com/file-extension/XMF",
```

Comment:
You don't need to include this, as it's not a primary source. The README also specifically calls out that "links to type aggregating sites and Wikipedia are not acceptable."

Reply:
Got it.